.travis.yml

```diff
@@ -26,7 +26,7 @@ env:
     - PIP_CACHE=$HOME/.cache/pip
     - SDK_TESTING=true
     - BUNDLE_PATH=$TRAVIS_BUILD_DIR/vendor/cache
-    - DD_AGENT_BRANCH=master
+    - DD_AGENT_BRANCH=jaime/pdhwheel
```

Comment:
Must be switched back to `master` after https://github.com/DataDog/dd-agent/pull/3691 is merged.